### PR TITLE
Moving operations list to its own page. Fixes #61

### LINKED ISSUE #61: Move operations list and maintenance to a new page
The page is getting too big. @brunodias20  suggested to make this a data table and I believe it great, but as this is a part of the page that we come very rarely, we could first just take it off the main page (reducing the amount of html sent) and then decide what to do with it.

### PR DESCRIPTION
**Pull request by danielfireman**
Can be checked at: isumm-canary.appspot.com

### CODE REVIEW

File: static/app.template.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <html>
 	<head>
+		<title>ISummary</title>
```
Comment (brunojdo) [inline static/app.template.html]: don't you think that is better maintain the app name? Or something like "Isumm - Summary". Just suggesting.

File: static/ops.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+function operationType(type){
```
Comment (brunojdo) [inline static/ops.js]: How about put js files into a js folder `/static/js` ?

Comment (danielfireman) [inline static/ops.js]: Done.

File: static/app.template.html (hunk 1 above)
Comment (danielfireman) [inline static/app.template.html]: Done.

**Comment by danielfireman** [comment]
Thanks a lot for the comments. All done. Please take another look. Thanks!

**Pull request merged**